util/config/run-in-venv.bash

```diff
@@ -3,8 +3,8 @@
 # usage: ./run-in-venv prog [args]
 
 if [ -z "$CHPL_HOME" ]; then
-  echo "Error: CHPL_HOME is not set" 1>&2
-  exit 1
+  # compute the chpl home directory
+  export CHPL_HOME=$(cd $(dirname $0) ; cd ..; cd ..; pwd)
```

Comment:
I don't think we need to export CHPL_HOME, I think we're just using it to find some scripts below. Should we instead us the `FIND_CHPL_HOME` style that the start_test/sub_test wrappers use?

Reply:
Other (Python) scripts we run currently require `CHPL_HOME` to be set. It seems better to have run-in-venv sort this out than having each python script run in the venv infer `CHPL_HOME` from its path.

In particular `make && make man` without `CHPL_HOME` set does not work without the export here.

Reply:
Hmm, how did those python scripts work before?

Reply:
The error we see without it is this:
```
Traceback (most recent call last):
  File "/home/mppf/w/10/doc/util/chpl2rst.py", line 312, in <module>
    main(**arguments)
  File "/home/mppf/w/10/doc/util/chpl2rst.py", line 288, in main
    preamble = gen_preamble(chapelfile, link=link)
  File "/home/mppf/w/10/doc/util/chpl2rst.py", line 132, in gen_preamble
    output.append(gen_link(link, chapelfile))
  File "/home/mppf/w/10/doc/util/chpl2rst.py", line 74, in gen_link
    chpl_home = os.path.realpath(os.getenv('CHPL_HOME'))
  File "/usr/lib/python3.8/posixpath.py", line 390, in realpath
    filename = os.fspath(filename)
TypeError: expected str, bytes or os.PathLike object, not NoneType
```

Reply:
It didn't work in the 1.23 release:

```
make -j11 && make docs
...
cd doc && make docs
Error: CHPL_HOME is not set
make[2]: *** [Makefile:125: error_docs] Error 1
make[1]: *** [Makefile:38: docs] Error 2
make: *** [Makefile.devel:26: docs] Error 2
```